prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -50,13 +50,12 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: alpine:3.15.0
+      - image: bash:alpine3.14
         command:
-        - /bin/sh 
+        - /usr/local/bin/bash
         args:
         - -c
-        - cd ./manifests/gcp_marketplace/test
-        - sh ./presubmit.sh
+        - cd ./manifests/gcp_marketplace/test && bash ./presubmit.sh
```

Comment:
nit: I think if the image has bash, then the entire cmd can be simplified as `./manifests/gcp_marketplace/test/presubmit.sh`, just like what we have for the rest tests, right?

Reply:
If it can be simplified then it is awesome, but the presubmit test calls the snapshot.sh in the same file, where it is taking the `${DIR}` parameter, if we call presubmit from root, the `${DIR}` value is root, which caused the snapshot.sh not found: https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/test/presubmit.sh#L38

Reply:
The DIR var is a common one-liner to find the folder containing the bash script, despite current working dir.
So you should be able to run it outside.

Reply:
I see, it is strange that ${DIR} doesn't work for me before, maybe it was because I was using /bin/sh instead of bash? Updated the command here.